lib/service/service.go

```diff
@@ -1194,6 +1194,8 @@ func NewTeleport(cfg *servicecfg.Config) (*TeleportProcess, error) {
 	// at any time with dynamic configuration
 	process.RegisterFunc("common.upload.init", process.initUploaderService)
 
+	process.RegisterFunc("update.deploy.agents", process.periodUpdateDeployServiceAgents)
+
```

Comment:
Can we add `aws-oidc` or something related?
So that it's easier to understand the context just from the method name